packages/frontend/src/helpers/lockTiles.test.jsx

```diff
@@ -0,0 +1,60 @@
+import { lockTiles, unlockTiles } from './lockTiles';
+import { createTilesFromPhotos } from './createTilesFromPhotos';
+import { mockPhotos } from '__mocks__/api/mockPhotos';
+
+describe('lockTiles', () => {
+  it('returns an array the length of the array it is given', () => {
+    const tiles = createTilesFromPhotos(mockPhotos);
+
+    const lockedTiles = lockTiles(tiles);
+
+    expect(lockedTiles.length).toEqual(tiles.length);
+  });
+
+  it('changes the is flippable value for each tile to false', () => {
+    const tiles = createTilesFromPhotos(mockPhotos);
+
+    const allTilesAreFlippable = tiles.some(
+      (tile) => tile.isFlippable === true
+    );
```

Comment:
I'm not sure this is checking what it seems to be checking. `some` returns `true` if at least one of the items in the array meets the given condition. If all tile are flippable, it'd have to be `!tiles.some(tile => tile.isFlippable === false)`, I think.

Reply:
Totally. Good call.